### ISSUE TITLE
Is it possible to search for an original contig id?

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1584]
![image](https://user-images.githubusercontent.com/7661533/28035231-856826ba-6579-11e7-878d-235f5e29f291.png)

**Comment by mshukla1** [comment]
Jim, 

Genomes are usually loaded with external database ids, i.e. genbank, refseq, biosample, bioproject, 
and assembly accessions, when available. 

This genome is has only genbank accessions in the database not refseq (NC_*). I suspect it is 
because, the genome may nit have been added at the time we picked it up from genbank. 

https://www.patricbrc.org/view/Genome/634458.3 <https://www.patricbrc.org/view/Genome/634458.3>

I will run a check on all public genomes and load missing db xrefs. 

-Maulik
